Polyfills/Canvas/Source/Context.cpp

```diff
@@ -650,15 +651,37 @@ namespace Babylon::Polyfills::Internal
             throw std::runtime_error{ "invalid argument" };
         }
 
-        const auto fontName = value.ToString();
-        if (m_fonts.find(fontName) == m_fonts.end())
+        // Default font id, and font size values.
+        // TODO: Determine better way of signaling to user that font specified is invalid.
+        m_currentFontId = -1;
+        int fontSize = 16;
+
+        // Parse user specified font style to set font ID and font size if specified.
+        std::istringstream fontOptionsStream(value.ToString());
+        std::string fontOption;
```

Comment:
Wouldn't it be easier (and more scalable to additional things like style as you mentioned) if it just used a regex?

Reply:
Good point, will look into using regex for pulling out arguments instead.

Reply:
Updated the PR to use regex for parsing tokens.  First time using C++ regex, so let me know if there's something smarter I could be doing instead.